internal/net/net.go

```diff
@@ -14,44 +14,20 @@
 package net
 
 import (
-	"context"
-	"io/ioutil"
-	"net/http"
-	"strings"
+	"github.com/mongodb/mongocli/internal/config"
```

Comment:
[q] should we keep this package? I feel we can be better off without and rely on our store package alone